core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -356,7 +407,9 @@ class PartKeyLuceneIndex(ref: DatasetRef,
     val document = makeDocument(partKeyOnHeapBytes, partKeyBytesRefOffset, partKeyNumBytes, partId, startTime, endTime)
     logger.debug(s"Upserting document ${partKeyString(partId, partKeyOnHeapBytes, partKeyBytesRefOffset)} " +
                  s"with startTime=$startTime endTime=$endTime into dataset=$ref shard=$shardNum")
-    indexWriter.updateDocument(new Term(PART_ID, partId.toString), document)
+    val doc = luceneDocument.get()
```

Comment:
This is same as ``document`` we have in previous line.

Reply:
Pls see other comment